templates/admin/cron.tmpl

```diff
@@ -24,7 +24,7 @@
 						<td>{{DateFmtLong .Next}}</td>
 						<td>{{if gt .Prev.Year 1 }}{{DateFmtLong .Prev}}{{else}}N/A{{end}}</td>
 						<td>{{.ExecTimes}}</td>
-						<td {{if ne .Status ""}}class="tooltip" data-content="{{.FormatLastMessage $.i18n.Language}}"{{end}} >{{if eq .Status "" }}—{{else if eq .Status "finished"}}{{svg "octicon-check" 16}}{{else}}{{svg "octicon-x" 16}}{{end}}</td>
+						<td {{if ne .Status ""}}class="tooltip" data-content="{{.FormatLastMessage $.i18n}}"{{end}} >{{if eq .Status "" }}—{{else if eq .Status "finished"}}{{svg "octicon-check" 16}}{{else}}{{svg "octicon-x" 16}}{{end}}</td>
```

Comment:
the name `$.i18n` maybe changed to `$.locale`?

Reply:
Seems like a good addition, but that would cost another 276 files to this PR(so should be another PR), I can make one after this is merged.

Reply:
https://github.com/go-gitea/gitea/pull/20153